src/statusBar.ts

```diff
@@ -31,15 +31,22 @@ class StatusBarClass implements vscode.Disposable {
     }
   }
 
-  public SetColor(color: string) {
+  public SetColor(background: string, foreground?: string) {
     const currentColorCustomizations = vscode.workspace
```

Comment:
Can you call `Configuration.getConfiguration` here?

Reply:
You mean instead of calling `vscode.workspace.getConfiguration('workbench').get('colorCustomizations')` I should use something like `Configuration.getConfiguration('workspace').get('colorCustomizations')`? (not able to look up the exact syntax right now :)). If so, then it should be used [when updating](https://github.com/mgor/Vim/blob/15ace4f2355ed1c624ed8868980263c5a73b3140/src/statusBar.ts#L52) the configuration as well?

Reply:
Actually scratch that, I realize this fx is not exposed in `IConfiguration`.